docs/docs/getting-started/componentDefinition.mdx

```diff
@@ -0,0 +1,13 @@
+A component is a Helm Chart that can be installed on an Airy Cluster in order to extend its functionality.
+
+Components can be characterized by the functionality they enable:
+
+1. UI Components that provide new user interfaces in Airy
+2. Connector Components that connect one or more third-party services with Airy to exchange data. 
+3. API Components that expose additional endpoints to Airy.
+
+:::note
+
+A Component can provide multiple functions (e.g., Connector & UI or UI & API) but usually has a single primary function. You can find a more detailed description of a specific component in the `description.yaml` file of the component in our [airy-components](https://github.com/airyhq/airy-components) respository.
```

Comment:
I agree that this is how we should do it (with the `description.yaml` file). However this is not there at the moment and the CI pipeline should be setup in place before we can write this.

Reply:
We will hold off on this detail in the definition and move it to the contributing components page. This is marked as under development, so will help to implement when we do this.